sdk/src/test/java/com/google/cloud/dataflow/sdk/io/bigtable/BigtableIOTest.java

```diff
@@ -517,6 +524,58 @@ public void testGetSplitPointsConsumed() throws Exception {
     reader.close();
   }
 
+  @Test
+  public void testAddBulkOptions() {
+    BigtableOptions.Builder optionsBuilder = BIGTABLE_OPTIONS.toBuilder();
```

Comment:
presumably set some non-defaults and then use your cloner - otherwise we can't tell whether or not things were actually cloned correctly.